src/css/components/all-categories.css

```diff
@@ -0,0 +1,109 @@
+/**
+  |============================
+  | All Categories
+  |============================
+*/
+
+.button {
+  background: none;
+  border-radius: 15px;
+  padding: 14px 28px;
+  font-weight: 500;
+  cursor: pointer;
+}
+
+.all-categories-button {
+  font-size: 18px;
+  line-height: calc(22 / 18);
+  color: var(--color-gray-pale-alpha050);
+  border-color: var(--color-gray-pale-alpha020);
+  border-radius: 15px;
+  border: 1px solid var(--color-gray-pale-alpha020);
+
+  margin-bottom: 30px;
+}
+
+.all-categories-button:hover,
+.all-categories-button:focus,
+.all-categories-button.is-active {
+  background-color: var(--color-accent-primary);
+  color: var(--color-gray-accent-primary);
+  border: 1px solid var(--color-accent-primary);
+  transition: all 0.3s;
```

Comment:
Change to `transition: background-color 250ms ease-in-out, color 250ms ease-in-out, border 250ms ease-in-out;`